images/win/scripts/Installers/Run-Antivirus.ps1

```diff
@@ -0,0 +1,39 @@
+################################################################################
+##  File:  Run-Antivirus.ps1
+##  Desc:  Run a full antivirus scan.
+##         Run right after cleanup before we sysprep
+################################################################################
+
+try{
+    Update-MpSignature
+}
+catch {
+    Write-Host "Some error was found"
```

Comment:
Please provide the current error that can be accessed using `$_` variable - https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_try_catch_finally?view=powershell-7

Reply:
Thank you, added Write-Host $_  into the catch block.